mlir/lib/Dialect/Rock/Transforms/AccelEmitter.cpp

```diff
@@ -236,11 +265,12 @@ ArrayAttr MfmaEmitter::computeOutputTransforms(PatternRewriter &b, Location loc,
   toMatrixC.embed(
       "gemmM", 1, matrixM,
       {"m", "wave_m", "m_tid", "m_i", "blk_row", "vec_group", "vec_item"},
-      {mPerBlock, mPerWave, rowGroupSize, mPerRepeat, m,
+      {mPerBlock, mPerAccel, rowGroupSize, mPerAccel * mWaves, m,
```

Comment:
These names are meaningfully different, and if you're going to change them, can you update `"wave_m"` to `"accel_m"` above as well?
And I think `mPerAccel * mWaves` should be `== mPerRepeat`?

Reply:
Why? We still need  `wave_m`. It's just the coefficient of the `embed` that is different. I mean, I am still looking at `wave_0`, `wave_1`, and so on, but their coefficient is different. 
- Before `wave_m` was mutiplied by `mPerWave`, because we were doing all the repeats together
- Now `wave_m` is multiplied by `mPerAccel`, because the i-th wave only computes `mPerAccel` and then "jump" of `mWaves`

Reply:
Ah. Makes sense.